oem/ibm/configurations/bios/enum_attrs.json

```diff
@@ -451,48 +451,60 @@
             "readOnly": true
         },
         {
-            "attribute_name": "pvm_keep_and_clear",
-            "possible_values": ["Disabled", "Enabled"],
+            "attribute_name": "hb_power_PS0_functional",
+            "possible_values": ["Enabled", "Disabled"],
             "default_values": ["Disabled"],
-            "helpText": "The hypervisor needs to clear most of PHYP NVRAM, but preserve the NVRAM for the manufacturing default partition",
-            "displayName": "pvm_keep_and_clear"
+            "helpText": "Specifies if Power Supply 0 is functional or not. (Enabled is Functional)",
+            "displayName": "Power Supply 0 is functional",
+            "dbus": {
+                "object_path": "/xyz/openbmc_project/inventory/system/chassis/motherboard/powersupply0",
+                "interface": "xyz.openbmc_project.State.Decorator.OperationalStatus",
+                "property_name": "Functional",
+                "property_type": "bool",
+                "property_values": [true, false]
+            }
         },
         {
-            "attribute_name": "pvm_clear_nvram",
-            "possible_values": ["Disabled", "Enabled"],
+            "attribute_name": "hb_power_PS1_functional",
```

Comment:
Same as above.
Shouldn't these attributes be added at the end and not modify existing attributes?

Reply:
See line 410 above.  This has something to do with what the tool thinks is happening.

Reply:
yes, looks like there were some duplicate entries